README.md

```diff
@@ -3,66 +3,171 @@
+
+* Stability
+* Ease of use / Practicability
+* Playability (ingame content)
+* Customization
+* Community driven software (check our discord)
+
+We also welcome new users (even non-english speaking users!) and help them learn / improve their skills (C++, SQL, Git, software collaboration, tutoring (wiki) etc...).
+
+Unlike other projects which focus more on the developer's side of things, we want users to be able to run their own server with as few troubles as possible. Our contributors all run their own private server (in local or not).
+
+That's why AzerothCore is easier to use, to maintain, to understand, to develop on and to customize to suit your needs, than other emulators.
+
+In short, we focus on the **user experience (UX)**, whether it be the **player's experience**, the **developer's experience** or the **administrator's experience**.
+
+
+## HOW TO THANK US
+
+Being an open source project, we rely on volunteers to pursue development. Here are ways to help us if you use AzerothCore:
+
+#### Github star
+
+Click on the "star this repository" button to help us gain more visibility on Github!
+
+#### By contributing
+
+Check the **CONTRIBUTING** section below.
+
+#### Financially :moneybag:
+
+You can support the project by financing the resolution of issues [using Bountysource](http://www.azerothcore.org/wiki/Bountysource).
+
+#### Advertising
+
+By talking about us on different platforms or to people who would like to get involved.
 
-For help requests, it is recommended to ask your question on [StackOverflow](https://stackoverflow.com/questions/tagged/azerothcore) and link it in [our chat](https://discordapp.com/channels/217589275766685707/284406375495368704).
 
-## Docker
+## CONTRIBUTING
+
+AzerothCore is a learning project, and there are lots of different ways to contribute to the project:
+
+* By testing our fixes (we can teach you how to correctly use Git to help us but that will also help you out tremendously)
+* By developing directly to the core or to the modules
+* By [creating new modules](http://www.azerothcore.org/wiki/Create-a-Module)
+* By improving our [wiki][wiki]
+* By providing direct support to our community (on discord, stackoverflow or specialized forums)
+* By making extra content (video tutorial for example)
+* By putting bounties on issues
+
+
+If you want to contribute to the project, you will find a lot of resources that will guide you in [our wiki][wiki].
+
+Feel free to join us on [our discord chat server][discord] where we taught a lot of new people how to get started and who are now important contributors!
+
+<!-- TO UNCOMMENT LATER -->
+<!-- As we put a big emphasis on community, there are also special rewards for contributors such as reputation ranks (displayed in our discord and our website), reputation badges (to display on your project/portfolio), premium software licenses, private modules access, private tools access and a lot of small private repositories access. -->
+
```

Comment:
?

Reply:
ah hmm i put this for later when we have a special page dedicated to this, lol